static/bidder-info/edge226.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "http://ssp.dauup.com/pserver"
+maintainer:
+  email: "audit@edge226.com"
+gvlVendorID: 1202
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+    
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://ssp.dauup.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
The redirect url is not working for me
https://ssp.dauup.com/pserver?gdpr=&gdpr_consent=&us_privacy=&redir=https%3A%2F%2Fib.adnxs.com%2Fsetuid%3Fbidder%3Dedge226%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%5BUID%5D

<img width="1663" alt="image" src="https://github.com/prebid/prebid-server/assets/87759626/b85b231e-5cf7-4120-a2c6-ccd1c3332686">

Reply:
@Edge226Ads The cookie sync url is still failing with 404. Please re-verify.